src/test/java/com/wikia/webdriver/common/dataprovider/TrackingOptInDataProvider.java

```diff
@@ -6,19 +6,18 @@
 
 public class TrackingOptInDataProvider {
 
-  private static final String
-      ADS_KIKIMORA_PATTERN =
-      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*";
+  private static final String ADS_KIKIMORA_OPT_IN_ACCEPTED_PATTERN =
+      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*u=0.*";
+  private static final String ADS_KIKIMORA_OPT_IN_REJECTED_PATTERN =
+      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*u=-1.*";
   private static final String ADS_KRUX_PATTERN = "https?://.*cdn\\.krxd\\.net.*";
-  private static final String
-      ADS_NETZ_ATHLETEN_PATTERN =
+  private static final String ADS_NETZ_ATHLETEN_PATTERN =
       "http?://.*adadapter\\.netzathleten-media\\.de/.*/naMediaAd\\.js.*";
-  private static final String ADS_MOAT_PATTERN = "https?://.*pixel\\.gif.*";
-  private static final String
-      ADS_PREBID_PATTERN =
+  private static final String ADS_MOAT_PATTERN = "https?://.*moatads\\.com.*";
+  private static final String ADS_PUBMATIC_PATTERN =
       "http?://.*gads\\.pubmatic\\.com/AdServer/AdCallAggregator.*";
   private static final String ADS_A9_PATTERN = "http?://.*amazon-adsystem\\.com/.*/apstag\\.js.*";
-  private static final String ADS_COMSCORE_PATTERN = "http?://.*scorecardresearch.*";
+  private static final String ADS_COMSCORE_PATTERN = "http?://.*scorecardresearch\\.com/beacon\\.js*";
   private static final String ADS_QUANTCAST_PATTERN = "http?://.*quantserve.*";
   private static final String ADS_APP_NEXUS_PATTERN = "http?://.*ib\\.adnxs\\.com/jpt.*";
   private static final String ADS_NORDICS_PATTERN = "https?://.*Nordics_RoN.*";
```

Comment:
This will match also `https://whateverorsomethingNordics_RONsomethingelse.edu.pl`. How about making sure the `Nordics_RoN` part is in a URL within DFP domain?